Presentations/03-data-wrangling.Rmd

```diff
@@ -129,6 +130,21 @@ In this session, you'll be introduced to some basic concepts of data cleaning in
 
 ---
 
+# Introduction
+
+### Before we start, let's make sure we are all set: 
+
+1. Start a fresh RStudio session.
+2. Open your RStudio project.
```

Comment:
I would add here that they can do both of these steps by double-clicking in the RStudio project file they create in the session before